### PR TITLE
Feature/image-group

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**

### CODE REVIEW

File: src/content/docs/guidelines/assets.mdx [hunk 1]
```diff
@@ -45,6 +45,12 @@ For all aspect ratios the images scale proportionally so they will not distort.
 
 These placeholder images can help visualise the image proportions and determine a focal point and ideal file size. It is recommended to use the WebP file type for optimal quality and size.
 
+#### `360x450px`
```
Comment (andij) [inline src/content/docs/guidelines/assets.mdx:48]: I'd prefer this placeholder not to be the first asset in this list (as it's rather unique).

I suggest moving it between 640x360px and 640x480px.

File: src/content/docs/components/ns-media-group.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+#### Specification notes
+
+##### `media` attribute
+The structure of the each of `media` object.
+
+| Key | Options | Description |
+| :--- | :--- |:--- |
+|`type`| `image` | The type of the media that needs to be displayed. |
+|`src`| Refer [Assets](/guidelines/assets) for choice of image sizes. | The link to the media. |
+|`ratio`| `1 / 1` `4 / 3` `4 / 5` `16 / 9` | The aspect ratio of the media. |
```
Comment (andij) [inline src/content/docs/components/ns-media-group.mdx:58]: Should we display all ratios? There's a 'dont - Change the ratio of the images' in the Guidance section.

File: src/data/badge.json [hunk 3]
```diff
@@ -11,6 +11,10 @@
     {
       "name": "ns-date-time",
       "badge": "new"
+    },
```
Comment (andij) [inline src/data/badge.json:14]: Please remove the oldest `new` badge from either `ns-date-time` or `ns-action-link`.

File: package.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "@astrojs/markdown-remark": "^4.2.0",
     "@astrojs/starlight": "^0.15.0",
     "@astrojs/starlight-docsearch": "^0.1.0",
-    "@connectedhomes/nucleus": "^3.5.0",
+    "@connectedhomes/nucleus": "^3.7.0",
```
[the remark this answers is not in the document]
Comment (MekalaNagarajan-Centrica) [reply package.json:18]: This will be automatically done when this is merged with main.

File: src/data/badge.json (hunk 3 above)
Comment (MekalaNagarajan-Centrica) [reply src/data/badge.json:14]: We need to check with the team for how long the badges should be present.

File: src/content/docs/components/ns-media-group.mdx (hunk 2 above)
Comment (MekalaNagarajan-Centrica) [reply src/content/docs/components/ns-media-group.mdx:58]: The component supports all the ratios.

File: src/content/docs/guidelines/assets.mdx (hunk 1 above)
Comment (MekalaNagarajan-Centrica) [reply src/content/docs/guidelines/assets.mdx:48]: The placeholder images are listed by ascending value of the dimension.

**Pull request merged**